aws-storage-s3/build.gradle

```diff
@@ -30,5 +30,11 @@ dependencies {
     testImplementation dependency.mockito
     testImplementation dependency.robolectric
     testImplementation dependency.androidx.test.core
+
+    androidTestImplementation project(path: ':testutils')
+    androidTestImplementation dependency.androidx.annotation
+    androidTestImplementation dependency.androidx.test.runner
+    androidTestImplementation dependency.androidx.test.junit
+    androidTestImplementation dependency.aws.mobileclient
```

Comment:
We already have mobile client as a dependency above - do we still need to include it as a test implementation?

Reply:
oof good catch. Will remove.